install/linux/usr/share/odemis/sim/sparc2-4spec-sim.odm.yaml

```diff
@@ -311,7 +352,7 @@ SPARC2-4Spec: {
 # In reality, this is a Zyla, but you need libandor3-dev to simulate an AndorCam3
 "AR Camera": {
     class: andorcam2.AndorCam2,
-    role: ccd0,
+    role: ccd,
```

Comment:
Is this change (and spectrometer0) necessary? Normally ccd and ccd0 should be equivalent.